docs/resources/logdna_alert.md

```diff
@@ -0,0 +1,117 @@
+# logdna_alert Resource
+
+Manages [LogDNA alert presets](https://docs.logdna.com/docs/alerts). To get started, all you need to do is to specify a `name` and configuration for one of our currently supported alerts- email, webhook, or PagerDuty.
+
+## Example - Basic Alert Setup
+
+```hcl
+provider "logdna" {
+  servicekey = "xxxxxxxxxxxxxxxxxxxxxxxx"
+  url = "https://api.logdna.com" # (Optional) specify a LogDNA Region
+}
+
+resource "logdna_alert" "my_alert" {
+  name = "My Alert Preset via Terraform"
+  email_channel {
+    emails          = ["test@logdna.com"]
+    immediate       = "false"
+    operator        = "presence"
+    triggerlimit    = 15
+    triggerinterval = "15m"
+    terminal        = "true"
+    timezone        = "Pacific/Samoa"
+  }
+}
+
+
+```
+## Example - In-Depth Alert Setup
+
+```hcl
+provider "logdna" {
+  servicekey = "xxxxxxxxxxxxxxxxxxxxxxxx"
+  url = "https://api.logdna.com" # (Optional) LogDNA Region Instance, needed for IBM-based instances
+}
+
+resource "logdna_alert" "my_alert" {
+  name = "Terraform Alert Preset In-Depth Example"
+  email_channel {
+    emails          = ["test@logdna.com"]
+    immediate       = "false"
+    operator        = "absence"
+    terminal        = "true"
+    timezone        = "Pacific/Samoa"
+    triggerinterval = "15m"
+    triggerlimit    = 15
+  }
+
+  pagerduty_channel {
+    immediate       = "true"
+    key             = "your pagerduty service key goes here"
+    terminal        = "true"
+    triggerinterval = "15m"
+    triggerlimit    = 15
+  }
+
+  webhook_channel {
+    bodytemplate = jsonencode({
+      message = "Alerts from {{name}}"
+    })
+    headers = {
+      "Authentication" = "auth_header_value"
+      "HeaderTwo"      = "ValueTwo"
+    }
+    immediate       = "false"
+    method          = "post"
+    terminal        = "true"
+    triggerinterval = "15m"
+    triggerlimit    = 15
+    url             = "https://yourwebhook/endpoint"
+ }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `name`: (Required) The name this Alert Preset will be given, type _string_
```

Comment:
@dm36, for alert presets, it's not just `name` but also there must (!) be, at least, 1 channel specified - need to document that as well.

Reply:
this is documented at the top of logdna_alert.md